docs/API/ConfigurationFile.md

```diff
@@ -34,6 +34,9 @@ methods.
 
 ## Updating ConfigurationFiles
 
+When you just want to add a new option to the config, you can simply add it to the "_config.yml_" resource file. It will 
```

Comment:
This is an API, that can handle every configuration file, not only the `config.yml` file